src/tests/test_piranha_java.rs

```diff
@@ -57,7 +58,6 @@ create_rewrite_tests! {
     }, cleanup_comments = true , delete_file_if_empty= false;
   test_scenarios_find_and_propagate:  "find_and_propagate", 2, substitutions = substitutions! {"super_interface_name" => "SomeInterface"},  delete_file_if_empty= false;
   test_non_seed_user_rule:  "non_seed_user_rule", 1, substitutions = substitutions! {"input_type_name" => "ArrayList"};
-  test_new_line_character_used_in_string_literal:  "new_line_character_used_in_string_literal",   1;
```

Comment:
Any reason why we are removing this? I don't see a contradiction between testing this scenario normally (for whichever feature it was already originally testing) and then also picking it to test the new code snippet functionality.